Gemfile

```diff
@@ -26,9 +26,6 @@ gem 'oily_png', '~> 1.1.1'
 gem 'nokogiri', '~> 1.5.10'
 gem 'therubyracer', platforms: :ruby, require: linux_only('therubyracer')
 gem 'aweplug', github: 'awestruct/aweplug'
-#gem 'aweplug', github: 'Dantheman720/aweplug', :branch => 'DEVELOPER-1248'
-#gem 'aweplug', path: '~/projects/ruby/aweplug'
-#gem 'aweplug', path: '../aweplug'
```

Comment:
These are useful during development, for when we need to point to a different Aweplug location. We have them commented out for convenience.